File: Java/Car.java
package Java;

public class Car {
    Integer id;
    String license;
    Account driver;
    private Integer passenger;

    public Car(String licence,Account driver){
        this.license = licence;
        this.driver = driver; 
    }

    void printDataCar(){
        if (passenger != null){
            System.out.println("Liecence: " + license + " Name Diver: " + driver.name + " Passengers: " + passenger);
        }
    }
 
    public Integer getPassenger(){
        return passenger;
    }
    
    public void setPassenger (Integer passenger){
        if (passenger == 4){
            this.passenger = passenger;
        }else{
            System.out.println("Necesitas asignar 4 pasajeros");
        }

    }
}

File: PHP/Car.php
<?php
class Car{
    public $id;
    public $license;
    public $driver;
    public $passenger;

    public function __construct($license,$driver){
        $this->licence = $license;
        $this->driver = $driver;
    }

    public function printDataCar(){
        echo "Licence: " . $this->licence . " Name Diver: " . $this->driver->name;
    }
}
?>